cmd/statreceiver/parser.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"log"
+	"sync"
+	"time"
+
+	"github.com/zeebo/admission/admproto"
+)
+
+// Parser is a PacketDest that sends data to a MetricDest
+type Parser struct {
+	d       MetricDest
+	f       []*PacketFilter
+	scratch sync.Pool
+}
+
+// NewParser creates a Parser. It sends metrics to d, provided they pass all
+// of the provided PacketFilters
+func NewParser(d MetricDest, f ...*PacketFilter) *Parser {
+	return &Parser{
+		d: d, f: f,
+		scratch: sync.Pool{
+			New: func() interface{} {
+				var x [10 * 1024]byte
```

Comment:
You can use `10 * memory.KB` to improve readability.

Reply:
!

Reply:
wait how? is there a memory package somewhere?

Reply:
storj.io/storj/internal/memory